.babelrc

```diff
@@ -1,10 +1,5 @@
 {
-  "presets": [
-    ["@babel/preset-env", {
-      "modules": false,
-      "targets": {
-        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
-      }
-    }]
```

Comment:
why did this change? (I don't actually know what it's for)

Reply:
From my understanding, `preset-env` is used to convert some of our JavaScript code written for `courseroad2` so that it actually works in browsers (some later JavaScript features vary greatly in terms of browser support). `transform-es2015-modules-commonjs` is used to convert JavaScript code into `CommonJS` (the module system used for requiring packages in things like `Node.js`.

From my understanding, we need to keep `preset-env` around, and I'm not sure why `transform-es2015-modules-commonjs` is needed unless something in `vue-cli` requires it. @georgiashay Do you know which version of `vue-cli` you're using so I can look into it?

Reply:
Added back in after much hassle.  Weird babel errors https://github.com/facebook/jest/issues/6913